### ISSUE TITLE
docker-compose up no longer works

### ISSUE THREAD

**Issue by epot** [bitnami/containers#55274]
### Name and Version

bitnami/kafka:latest

### What architecture are you using?

amd64

### What steps will reproduce the bug?

I have this `docker-compose.yml` file that was working until earlier today:

```yaml
version: "3"
services:
  kafka:
    image: 'bitnami/kafka:latest'
    ports:
      - '9095:9095'
    environment:
      - KAFKA_CFG_NODE_ID=0
      - KAFKA_CFG_PROCESS_ROLES=controller,broker
      - KAFKA_CFG_LISTENERS=PLAINTEXT://:9092,CONTROLLER://:9093
      - KAFKA_CFG_LISTENER_SECURITY_PROTOCOL_MAP=CONTROLLER:PLAINTEXT,PLAINTEXT:PLAINTEXT
      - KAFKA_CFG_CONTROLLER_QUORUM_VOTERS=0@kafka:9093
      - KAFKA_CFG_CONTROLLER_LISTENER_NAMES=CONTROLLER
```

that I run locally on my mac with `docker-compose up`.

### What is the expected behavior?

To run a kafka locally.

### What do you see instead?

I see this error:

```console
$ docker-compose up
[+] Building 0.0s (0/0)                                                                                                                                                                     docker:desktop-linux
[+] Running 1/0
 ✔ Container apache_kafka_compare-kafka-1  Recreated                                                                                                                                                        0.0s
Attaching to apache_kafka_compare-kafka-1
apache_kafka_compare-kafka-1  | kafka 16:55:56.65 INFO  ==>
apache_kafka_compare-kafka-1  | kafka 16:55:56.65 INFO  ==> Welcome to the Bitnami kafka container
apache_kafka_compare-kafka-1  | kafka 16:55:56.65 INFO  ==> Subscribe to project updates by watching https://github.com/bitnami/containers
apache_kafka_compare-kafka-1  | kafka 16:55:56.65 INFO  ==> Submit issues and feature requests at https://github.com/bitnami/containers/issues
apache_kafka_compare-kafka-1  | kafka 16:55:56.65 INFO  ==>
apache_kafka_compare-kafka-1  | kafka 16:55:56.66 INFO  ==> ** Starting Kafka setup **
apache_kafka_compare-kafka-1  | kafka 16:55:57.56 WARN  ==> Kafka has been configured with a PLAINTEXT listener, this setting is not recommended for production environments.
apache_kafka_compare-kafka-1  | kafka 16:55:57.57 INFO  ==> Initializing Kafka...
apache_kafka_compare-kafka-1  | kafka 16:55:57.57 INFO  ==> No injected configuration files found, creating default config files
apache_kafka_compare-kafka-1  | kafka 16:55:57.64 INFO  ==> Initializing KRaft storage metadata
apache_kafka_compare-kafka-1  | kafka 16:55:57.64 WARN  ==> KAFKA_KRAFT_CLUSTER_ID not set - If using multiple nodes then you must use the same Cluster ID for each one
apache_kafka_compare-kafka-1  | kafka 16:55:58.43 INFO  ==> Generated Kafka cluster ID 'yL5Chu4ARo6-VYZdxdVS2w'
apache_kafka_compare-kafka-1  | kafka 16:55:58.44 INFO  ==> Formatting storage directories to add metadata...
apache_kafka_compare-kafka-1  | Exception in thread "main" org.apache.kafka.common.config.ConfigException: Invalid value  for configuration message.max.bytes: Not a number of type INT
apache_kafka_compare-kafka-1  | 	at org.apache.kafka.common.config.ConfigDef.parseType(ConfigDef.java:745)
apache_kafka_compare-kafka-1  | 	at org.apache.kafka.common.config.ConfigDef.parseValue(ConfigDef.java:493)
apache_kafka_compare-kafka-1  | 	at org.apache.kafka.common.config.ConfigDef.parse(ConfigDef.java:486)
apache_kafka_compare-kafka-1  | 	at org.apache.kafka.common.config.AbstractConfig.<init>(AbstractConfig.java:112)
apache_kafka_compare-kafka-1  | 	at org.apache.kafka.common.config.AbstractConfig.<init>(AbstractConfig.java:145)
apache_kafka_compare-kafka-1  | 	at kafka.server.KafkaConfig.<init>(KafkaConfig.scala:1601)
apache_kafka_compare-kafka-1  | 	at kafka.server.KafkaConfig.<init>(KafkaConfig.scala:1603)
apache_kafka_compare-kafka-1  | 	at kafka.tools.StorageTool$.$anonfun$main$1(StorageTool.scala:50)
apache_kafka_compare-kafka-1  | 	at scala.Option.flatMap(Option.scala:271)
apache_kafka_compare-kafka-1  | 	at kafka.tools.StorageTool$.main(StorageTool.scala:50)
apache_kafka_compare-kafka-1  | 	at kafka.tools.StorageTool.main(StorageTool.scala)
apache_kafka_compare-kafka-1 exited with code 1
```

### Additional information

I tried downgrading to an earlier version since I saw a new release today, but I got the same issue.

**Comment by colatkinson** [comment]
Can confirm I'm also affected by this.

As a workaround, explicitly setting `KAFKA_CFG_MESSAGE_MAX_BYTES=1048588` ([the default value](https://kafka.apache.org/documentation/#brokerconfigs_message.max.bytes)) seems to do the trick.

**Comment by Dynee** [comment]
Confirming also affected. It looks like the [kafka-env.sh](https://github.com/bitnami/containers/blame/main/bitnami/kafka/3.6/debian-11/rootfs/opt/bitnami/scripts/kafka-env.sh#L40-L42) script was modified to include this env var with the recent release ~5 hours ago.

**Comment by yves-tutti** [comment]
Please review the fix in https://github.com/bitnami/containers/pull/55276

**Comment by juan131** [comment]
Fixed at https://github.com/bitnami/containers/pull/55276 thanks to @yves-tutti 

The fix was included in `bitnami/kafka:3.6.1-debian-11-r3` version.
